tests/prepare/freeze/test.sh

```diff
@@ -8,7 +8,7 @@ rlJournalStart
     rlPhaseEnd
     rlPhaseStartTest
         # Timeout large enough to boot the VM
-        rlRun "rlWatchdog \"rlRun 'tmt run -vfi $tmp' 2\" 60"
+        rlRun "rlWatchdog \"rlRun 'tmt run -vfi $tmp' 2\" 120"
```

Comment:
Does it even need to have a timeout? Wouldn't `rlRun "tmt run -vfi $tmp" 2` work as well?

Reply:
The thing is that if `tmt` is not able to handle the freeze (which the test is verifying) we would get the whole test stuck here. So, I believe, simple `rlRun` is not enough here.